features/login.feature

```diff
@@ -0,0 +1,75 @@
+Feature: Login behavior
+@QA
+Scenario: Test login logic: error handling, demo mode, and normal behavior
+    Then I install the ccz app at "test_list_search.ccz"
+
+    # normal login
+    Then I login with username "user_with_no_data" and password "123"
+    # TODO: consider counting the home screen buttons
+    Then I verify that all home buttons are present
```

Comment:
Can you call this after each login? I know it might seem like overkill but there also isn't any reason not to, and it keeps with the original test plan